cluster/addons/node-problem-detector/npd.yaml

```diff
@@ -76,6 +76,7 @@ spec:
       - name: localtime
         hostPath:
           path: /etc/localtime
+          type: "FileOrCreate"
```

Comment:
Is this right?  Why OrCreate?  @Random-Liu what happens if this file is not present in the host and becomes empty for NPD?